Gemfile

```diff
@@ -43,7 +43,7 @@ end
 # since that's not expressible here, we make it >= the last *known* version to
 # at least prevent downgrades beyond that:
 group(:omnibus_package) do
-  gem "appbundler", github: "chef/appbundler" # until next release with multiple-gem support
+  gem "appbundler", git: "https://github.com/chef/appbundler" # until next release with multiple-gem support
```

Comment:
the way i have my .gitconfig setup breaks if i don't fix these.  wish bundler would fix it, because i like the syntax, but they feel its a major breaking change.